broker/tests/unit/plugin/kubejobs/test_plugin.py

```diff
@@ -17,7 +17,8 @@
 import requests_mock
 import json
 import threading
-
+import copy
```

Comment:
pep8: respect alphabetical order of imports and group them by type as per [1]. 

[1] https://www.python.org/dev/peps/pep-0008/#id23